src/locales/en/messages.po

```diff
@@ -449,15 +449,15 @@ msgstr "Borrow"
 msgid "Borrow APY rate"
 msgstr "Borrow APY rate"
 
-#: src/modules/markets/Gho/GhoBanner.tsx
-msgid "Borrow APY, fixed rate"
-msgstr "Borrow APY, fixed rate"
-
 #: src/modules/markets/MarketAssetsList.tsx
 #: src/modules/markets/MarketAssetsListMobileItem.tsx
 msgid "Borrow APY, stable"
 msgstr "Borrow APY, stable"
 
+#: src/modules/markets/Gho/GhoBanner.tsx
+msgid "Borrow APY, stepped rate"
```

Comment:
This translation should be re-run, doesn't match the text in GhoBanner

Reply:
fixed